qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/EditDemographicsAndContactInfoSteps.java

```diff
@@ -119,17 +159,14 @@ public void editContactInfo() {
 		registrationEditSectionPage.enterPostalCode(testPatient.postalCode);
 		registrationEditSectionPage.clickOnPhoneNumberEdit();
 		registrationEditSectionPage.enterPhoneNumber(testPatient.phone);
-	}
-
-	@And("a user clicks on the confirm button")
-	public void saveUpdatedPatientDetails() throws InterruptedException {
+		assertTrue(registrationEditSectionPage.getInvalidPhoneNumberNotification().contains("Must be a valid phone number (with +, -, numbers or parentheses)"));
+		registrationEditSectionPage.clearPhoneNumber();
+		registrationEditSectionPage.enterPhoneNumber("111111111");
```

Comment:
aren't these call methods supposed to be under `editContactInfo()`
```js
assertTrue(registrationEditSectionPage.getInvalidPhoneNumberNotification().contains("Must be a valid phone number (with +, -, numbers or parentheses)"));
registrationEditSectionPage.clearPhoneNumber();
registrationEditSectionPage.enterPhoneNumber("111111111");
```

Reply:
not quite sure if i got your question clearly, care to elaborate or if im to answer as i understood it, the methods in question are under `editContactInfo()` as you have asked. you can a look-see at https://github.com/openmrs/openmrs-contrib-qaframework/blob/a67110b9cb88d15e26c3ed2ff8db2a8013d275b3/qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/EditDemographicsAndContactInfoSteps.java#L146

Reply:
could you take a close look at what saveUpdatedPatientDetails() is actually performing?